<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>780</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_InputsAndControls">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Inputs</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max Retries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tx_MaxRetries">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Delay Between Retires (Seconds)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tx_Delay">
       <property name="text">
        <string>0.05</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Controls</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_SelectFolder">
       <property name="text">
        <string>Select Folder To Save Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_BrowseFolder">
       <property name="text">
        <string>Browse Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_SX">
       <property name="text">
        <string>SX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_S1">
       <property name="text">
        <string>S1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_S2">
       <property name="text">
        <string>S2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_F1">
       <property name="text">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_F2">
       <property name="text">
        <string>F2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_F3">
       <property name="text">
        <string>F3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_StartCamera">
       <property name="text">
        <string>Start Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_StartProcess">
       <property name="text">
        <string>Start Processing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_StopProcess">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Processing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_StopCamera">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_TestRS485">
       <property name="text">
        <string>Test RS485</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>860</width>
      <height>460</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>850</width>
          <height>640</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>860</width>
      <height>320</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_Outputs">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Outputs</string>
       </property>
       <widget class="QTextEdit" name="tE_Outputs">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>20</y>
          <width>751</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
